test/integration/monitoring/test_timeseries.rb

```diff
@@ -151,6 +151,9 @@ def test_multiple_timeseries
         :page_size => 1
       )
     end
+
+    mu_pp_for_diff(resp)
```

Comment:
Do you intentionally want to leave this here or is it a leftover from debugging?

Reply:
Leftover. Will remove.